packages/fx-core/src/plugins/resource/frontend/dotnet/constants.ts

```diff
@@ -85,3 +88,14 @@ export class DotnetConfigInfo {
   static readonly webAppDomain = "domain";
   static readonly projectFilePath = "projectFilePath";
 }
+
+export class WebappBicepFile {
+  static readonly configurationTemplateFileName = "webappConfiguration.template.bicep";
+  static readonly provisionTemplateFileName = "webappProvision.template.bicep";
+}
+
+export class WebappBicep {
+  static readonly webappEndpoint = "provisionOutputs.webappOutput.value.endpoint";
```

Comment:
Is it possible to make it as a constant so that if the configuration is changed, we only update one place in the code.